plaso/parsers/logd.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""Parser for the logd.0.log file obtained from iOS Sysdiagnose file dumps."""
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class LogdEventData(events.EventData):
```

Comment:
I suggest we change LogEventData to something more concrete like IOSysdiagnoseLogData or similar.